### PR TITLE
Code changes to fix the description bug

### PR DESCRIPTION
**Pull request by Shwetap05**
If description of the parameter is not the last node e.g in below, then "Request Body Example" will be used as description instead of Request Body

<param name="sampleObject1" in="body"><see cref="T:System.String"/>Request Body
        <example><value>Request Body Example</value></example>
</param>

### CODE REVIEW

File: test/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.Tests/OpenApiDocumentGeneratorTests/Input/Annotation.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
       <group>Sample V1</group>
       <verb>GET</verb>
       <url>http://localhost:9000/V1/samples/{id}?queryBool={queryBool}</url>
-      <param name="sampleHeaderParam1" cref="T:Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.Tests.Contracts.SampleObject1" in="header">Header param 1</param>
```
Comment (Shwetap05) [inline test/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.Tests/OpenApiDocumentGeneratorTests/Input/Annotation.xml]: >"heade [](start = 136, length = 6)

unintentional change will revert

**Comment by Shwetap05** [comment]
Mentioned it in description, is that not clear?

---
In reply to: [403985847](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/140#issuecomment-403985847) [](ancestors = 403985847)

**Comment by scott-lin** [comment]
I read the description, but I don't get it. Is there an issue or bug I can read? Or can you give an example of what is correct vs not correct?

**Comment by Shwetap05** [comment]
I didn't log an issue but sure can if it helps, but content will not change much so explaining here, consider below documentation

<param name="sampleObject1" in="body"><see cref="T:System.String"/>Request Body
        <example><value>Request Body Example</value></example>
</param>

In the above example description of the param "sampleObject1" is Request Body but before this fix, "Request Body Example" is being parsed as description which is a bug. And that is happening b/c paramElement.DescendantNodes().ToList() returns
list of all the descendants node and not just the immediate child node, and then the logic was picking last text node as description which was Request Body Example and not Request Body. So updated code to use 
element.Nodes() which gives immediate child node and fixes the issue. Does this make issue more clear?



---
In reply to: [403991798](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/140#issuecomment-403991798) [](ancestors = 403991798)

**Pull request merged**